lib/linked_in/client.rb

```diff
@@ -5,13 +5,15 @@ module LinkedIn
   class Client
     include Helpers::Request
     include Helpers::Authorization
+    include Helpers::V2Request
     include Api::QueryHelpers
     include Api::People
     include Api::Groups
     include Api::Companies
     include Api::Jobs
     include Api::ShareAndSocialStream
     include Api::Communications
+    include Api::V2
```

Comment:
oh I see why you prefixed the methods now...

hmmm... well I suppose that's ok it just feels a bit awkward to me is all.